apps/helpthechickens.com/src/app/page.tsx

```diff
@@ -61,21 +63,247 @@ const Hero = () => {
+      <h2 className="border-b border-slate-300 pb-2 uppercase text-xl tracking-wide text-slate-800">
+        Years of evidence show a consistent pattern of extreme animal cruelty.
+      </h2>
+      <div className="flex flex-col lg:flex-row items-center lg:items-start justify-evenly gap-12 text-center">
+        <div className="flex flex-col gap-6 max-w-xs w-full">
+          <img
+            src='/img/criminal-neglect-of-chicken.png'
+            className="rounded-full size-96 object-cover object-top"
+            alt="Neglect of chickens"
+          />
+          <div className="flex flex-col gap-4">
+            <div className="text-4xl uppercase font-semibold">Criminal Neglect</div>
+            <p>
+              Sonoma County Animal Services examined birds from a Petaluma
+              Poultry factory farm in Petaluma in 2018 and identified wing and
+              joint injuries, open sores, and necrotic wounds so deep that
+              muscle and bone were exposed. They referred the owner of the
+              facility as a suspect for animal cruelty to the Sheriff&apos;s Office,
+              but charges were never filed.
+            </p>
+          </div>
+        </div>
+        <div className="flex flex-col gap-6 max-w-xs w-full">
+          <img
+            src='/img/chicken-zoonotic-pathogens.png'
+            className="rounded-full size-96 object-cover"
+            alt="Zoonotic Pathogens"
+          />
+          <div className="flex flex-col gap-4">
+            <div className="text-4xl uppercase font-semibold">Zoonotic Pathogens</div>
+            <p>
+              Lab testing has identified numerous infectious pathogens and
+              diseases present at Petaluma Poultry factory farms in Sonoma
+              County, including Infectious Bursal Disease, Infectious Bronchitis
+              Virus, and a highly antibiotic-resistant Enterococcus bacteria.
+            </p>
+          </div>
+        </div>
+        <div className="flex flex-col gap-6 max-w-xs w-full">
+          <img
+            src='/img/chickens-boiled-alive-slaughterhouse-1.png'
+            className="rounded-full size-96 object-cover"
+            alt="Chickens boiled alive at slaughterhouse"
+          />
+          <div className="flex flex-col gap-4">
+            <div className="text-4xl uppercase font-semibold">Animals Boiled Alive</div>
+            <p>
+              Chickens are stunned and slaughtered in the dark at high speeds,
+              so many of them miss the stunning stage. This means they can enter
+              scalding hot water while still conscious. A whistleblower
+              documented bright red chicken parts that had been discarded inside
+              the Petaluma Poultry slaughterhouse, an indication that the birds
+              were scalded alive and subsequently condemned.
+            </p>
+          </div>
+        </div>
+      </div>
+    </Section>
+  );
+};
+
+function FullInvestigatoryReport() {
```

Comment:
suddenly hate tailwind? 😄

Reply:
jk i know this is old

Reply:
haha. yeah i want to keep them identical so it's easier to fix them both later...someday